### PR TITLE
Add more badges and contributors list

### LINKED ISSUE #264: Make README great again

### PR DESCRIPTION
**Pull request by zamotany**
Closes #264 

@grabbou @thymikee @Krizzu @satya164 please check if your contribution types are correct.

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+        "code"
+      ]
+    },
+    {
+      "login": "orta",
+      "name": "Orta",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/49038?v=4",
+      "profile": "http://orta.io",
+      "contributions": [
+        "code"
```
Comment (thymikee) [inline .all-contributorsrc]: should be `"doc"`

File: .all-contributorsrc [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+        "doc",
+        "example",
+        "ideas"
+      ]
+    },
+    {
+      "login": "Krizzu",
+      "name": "Krzysztof Borowy",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/6444719?v=4",
+      "profile": "http://krizzu.github.io",
```
Comment (Krizzu) [inline .all-contributorsrc]: https://twitter.com/Krizzu

Comment (Krizzu) [inline .all-contributorsrc]: https://twitter.com/Krizzu

Comment (Krizzu) [inline .all-contributorsrc]: https://twitter.com/Krizzu
Please :)

File: .all-contributorsrc [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+        "example",
+        "ideas",
+        "review"
+      ]
+    },
+    {
+      "login": "satya164",
+      "name": "Satyajit Sahoo",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/1174278?v=4",
+      "profile": "https://medium.com/@satya164",
```
Comment (satya164) [inline .all-contributorsrc]: Twitter link please

File: .all-contributorsrc [hunk 4]
```diff
@@ -0,0 +1,345 @@
...
+    "README.md"
+  ],
+  "imageSize": 100,
+  "commit": false,
+  "contributors": [
+    {
+      "login": "grabbou",
+      "name": "Mike Grabowski",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/2464966?v=4",
+      "profile": "https://github.com/grabbou",
```
Comment (grabbou) [inline .all-contributorsrc]: I would also request twitter to be here :D

**Comment by grabbou** [comment]
LGTM - I think just missing meta for few folks and we can land this!

**Pull request merged**